dwave/optimization/src/nodes/mathematical.cpp

```diff
@@ -1692,6 +1692,10 @@ template <>
 bool UnaryOpNode<functional::square<double>>::integral() const {
     return array_ptr_->integral();
 }
+template <>
+bool UnaryOpNode<functional::square_root<double>>::integral() const {
+    return array_ptr_->integral();
```

Comment:
This is saying that if `a` is an integral than so shall `sqrt(a)`, which is not true. We should return `false`.